packages/shared/src/hooks/useMutateFilters.ts

```diff
@@ -423,27 +515,19 @@ export default function useMutateFilters(
     [user, queryClient, feedId],
   );
 
-  const { mutateAsync: unfollowSourceRemote } = useMutation<
-    unknown,
-    unknown,
-    SourceMutationProps,
-    () => Promise<void>
-  >(
-    ({ source }) =>
-      gqlClient.request(ADD_FILTERS_TO_FEED_MUTATION, {
-        filters: {
-          excludeSources: [source.id],
+  const blockSourceRemote = useCallback(
+    async ({ source }: SourceMutationProps) => {
+      await addFiltersRemote(
+        { source, key: 'excludeSources' },
```

Comment:
Can we make this static and two mutations? The `key` pattern seems a bit fragile and also not typed.

Reply:
@capJavert updated ✅

Reply:
Nice!